src/components/InputPeople/InputPeople.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect } from 'react';
+import { Person } from '../../types/Person';
+
+interface Props {
+  query: string;
+  appliedQuery: string;
+  setAppliedQuery: (string: string) => void;
+  handleQueryChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  setQuery: (string: string) => void;
+  selectedPeople: Person | null;
+  setSelectedPeople: (person: Person | null) => void;
+  // onFocus: (event: React.FocusEvent<HTMLInputElement, Element>) => void;
+  showList: boolean;
+  setShowList: React.Dispatch<React.SetStateAction<boolean>>;
+  setPageTitle: (string: string) => void;
```

Comment:
You can improve the structure of props by moving all values to the top and functions to the bottom.